recipes/osx-sysroot/meta.yaml

```diff
@@ -0,0 +1,78 @@
+{% set name = "osx-sysroot" %}
+{% set version = "1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+
+outputs:
+  - name: macosx-sdk
+    version: {{ _MACOSX_SDK_VERSION_ }}
+    script: build_sdk.sh
+    build:
+      noarch: generic
+      # weigh down newer sdks
+      # the default osx-64 package has 0 features
+      # alternatives for osx-64 have 1 feature
+      # default for osx-arm64 has 2 features
+      # alternatives for osx-arm64 have 3 features
+      # the system SDK has 4 features
+      track_features:                                       # [is_default_sdk == "no" or is_arm64 == "yes"]
+        - macosx-sdk_default_{{ _MACOSX_SDK_VERSION_ }}     # [is_default_sdk == "no"]
+        - macosx-sdk_arm1_{{ _MACOSX_SDK_VERSION_ }}        # [is_arm64 == "yes"]
+        - macosx-sdk_arm2_{{ _MACOSX_SDK_VERSION_ }}        # [is_arm64 == "yes"]
+        - macosx-sdk_system_{{ _MACOSX_SDK_VERSION_ }}      # [is_system_sdk == "yes"]
```

Comment:
I admit that I don't fully understand the design intention here (despite the comment), but using `[is_system_sdk == "yes"]` is incompatible with the selector for `track_features:`.